packages/storybook/stories/components/cart/migration.from.17.x.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="ODS Components/User feedback/Cart/Migration From 17.x" />
+
+# Cart - migrate from v17 to v18
+----
+
+Cart has been removed from ODS components. <img src="https://img.shields.io/badge/removed-FF0000" />
+
+This component was too specific to a custom need and comes with quite a lot of downsides.
+
+## Current component issues
+
+### Performance & Complexity
+
+The whole component was composed of 9 web-components, each with their own rendering lifecycle.
+
+Each of those were relying on DOM queries to update their states accordingly to their parents/siblings, which
+can become quite problematic performance-wise.
+
+Plus to use this component, integrators had to learn about the 9 different interfaces and all the possible combinations.
+
+### Internal logic
+
+ODS component tends to be presentational component, the internal logic should be handle by people integrating the component.
+
+Current Cart implementation was trying to enforce some of the logic (ex: quantity computation).
```

Comment:
The current implementation of Cart was trying to enforce some of the logic (e.g. quantity computation).